### PR TITLE
Hyperion, a building event map

### PR DESCRIPTION
**Pull request by png103**
## What Does This PR Do

I've been working on for a week at this point, many thanks to Syrrian for giving feedback throught and S34N for doing the piping/wiring

After nanotrasen's engineering team failed to meet time frames on the construction of their next station, Instead of canceling the shift and giving everyone the day off they chose to disguise the unfinished station as a "fun" company event where they'd just let whoever the hell wanted to finish the construction for them, isn't extra unpaid labor sweet?

The map only contains the bare minimum for everything to not spiral instantly into complete anarchy,

- A kitchen, so the crew doesn't starve to death
- Botany (or Hydroponics if you prefer so)
- Janitor's closet to clean up the inevitable messes
- Cargo to order the usual silly things
- Engineering supply closet to generate power
- Brig to give security something to do
- RnD to make electronics
- A table and two chairs for bridge hobos

There's a list of things that the crew is assigned to build, included in a note that an admin would spawn in bridge at round start

`[center][logo][/center][br]Dear capt.,[br]We regret to inform you that our team of skilled engineers failed to complete the station's construction in time, And as a result, you may have noticed that it is somewhat... incomplete.[br][br]So. after a few days of deliberation and planning, we are delighted to announce that Hyperion station has been cancelled, and on it's place, we present: project Build Your Own Nanotrasen Depot (B.Y.O.N.D.), Your goal is to encourage the crew to assist in the expansion and construction of the station; we have provided you with the necessary materials and tools, higger ups will be sent later to review your progress and give the final score. If you require anything specific, please contact central command, but keep in mind that we will only accept requests once.[br][br]The station necessitates the following:[br][br]-Rooms with proper wiring/lighting/air flow[br]-Clean and tidy departments[br]-A place for dining/drinking (preferably with actual food and drinks)[br]-A medical center[br]]-A waste disposal system[br]-A conference room[br][br][br]Extra points.[br][br]The features listed bellow are optional, however they will help improve your rating.[br][br]-A theatre or stage[br]-Dormitories[br]-A locker room[br]-A chapel or church[br]-A library[br]-A recreational room[br][br]So, without further ado, we wish you all the best of luck.[br][br][i]-Do not let us down!, Glory to Nanotrasen Inc![/i]`

I've ran the map several times, from what i've seen everything works as intended, but if any changes are needed i'll (try to) comply
## Why It's Good For The Game

Encouraging players to build stuff is fun
## Images of changes

Arrival shuttle (mining shuttle docking on the left and mini medbay on the right)
image
Station
image
Close ups
image
image
image
image
image

🆑 S34NW, Sirryan2002, png103
Add: new map
/ 🆑

### CODE REVIEW

**Comment by Sirryan2002** [comment]
Just to be clear to those unaware, if given the go ahead by maints/host, this will be TM'd once and then closed afterwards. This is not going to be a permanent feature addition.

**Comment by Denthamos** [comment]
I like the idea, though considering how high traffic the arrivals shuttle will be already due to it being a spawn in location and the only functional on site medical area at the start, if the mining area is in a bad spot.  Having where miners will be constantly going back and forth through, where the station personnel will constantly be going to in order to get metal and glass, and such will make that area a bit too congested for such uses.  

At the same time however not sure where else would be a good spot for it to dock at.

**Comment by png103** [comment]
> I like the idea, though considering how high traffic the arrivals shuttle will be already due to it being a spawn in location and the only functional on site medical area at the start, if the mining area is in a bad spot. Having where miners will be constantly going back and forth through, where the station personnel will constantly be going to in order to get metal and glass, and such will make that area a bit too congested for such uses.
> 
> At the same time however not sure where else would be a good spot for it to dock at.

i've placed down plenty of metal in the station, you can see it in the tables on the first close up, so the orm would probably only be used for emergencies when they are REALLY out, or just for plasma and all that so i bet it wouldn't be that crowded

**Comment by png103** [comment]
if a host/maintainer is reading this, i'm still working on improving the map so the one currently on the branch is not that final but it's pretty close

**Comment by S34NW** [comment]
Only minor mapping issues present, will be fine for an event shift.

**Review by S34NW — APPROVED** [review]

**Comment by hal9000PR** [comment]
For changelogs you can put the names of all that worked on it next to the first :cl:
Eg. 
🆑 S34N, sirryan, png103
Add: new map
/ 🆑

**Comment by png103** [comment]
alright so, i fixed some mistakes with the map, added an atmos storage and made this better, also added an art vendor so crew can get crayons and all
image
image
image

**Comment by Denthamos** [comment]
It would be neat to see how this works out otherwise for a planned event.  Throwing my approval on it as long as you properly coordinate a planned event time for this to happen and arrange for it to be announced on discord for players to know when the test merge and event is happening.

**Review by dearmochi — CHANGES_REQUESTED** [review]
Please revert the Cyberiad changes -- the map `hyperion` datum will be used for the admin to force switch to it when it's event time.

File: code/modules/mapping/cyberiad.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /datum/map/cyberiad
-	fluff_name = "NSS Cyberiad"
-	technical_name = "Cyberiad"
+	fluff_name = "Hyperion"
+	technical_name = "Project B.Y.O.N.D"
```
Comment (dearmochi) [inline code/modules/mapping/cyberiad.dm:3]:
```suggestion
	fluff_name = "NSS Cyberiad"
	technical_name = "Cyberiad"
```

This still needs reverting.

**Review by dearmochi — CHANGES_REQUESTED** [review]

**Comment by png103** [comment]
oops forgot about that

**Comment by png103** [comment]
borg recharging pod and like 5 rcds to avoid atleast a little bit of metal to be used up on the autolathes (also forgot to include some pictures of the cryo pods
image
image
image

File: code/modules/mapping/hyperion.dm [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/datum/map/hyperion
+	fluff_name = "Hyperion"
+	technical_name = "Project B.Y.O.N.D"
```
Comment (AffectedArc07) [inline code/modules/mapping/hyperion.dm:3]:
```suggestion
	technical_name = "hyperion"
```

**Comment by AffectedArc07** [comment]
image

**Comment by png103** [comment]
i made some changes to fix the z/w issues and also fixed some spawn issues i'm testing it rn before i add it to the branch

**Comment by png103** [comment]
i still have a changes requested warning despite doing the requested change
image

**Comment by png103** [comment]
by the way this isn't coding related but i need 2/3 admins that would play as the inspectors at the end of the round
also if you're gonna be on the event dm me (Peterr#9201) so i can send you a google docs with some basic midround events and timings

**Comment by png103** [comment]
that was really fun, i'm glad it went (mostly) well

**Pull request closed without merging**